Develop player object

gitonga13 opened the pull request:
Ticket:

#13

Testing:

.Built a player object with the player variables

gitonga13 commented on HW1_DiceGame/app/src/main/java/edu/fullerton/csu/team06/hw1_dicegame/Player.java:
```diff
@@ -2,13 +2,78 @@
+        this.name = name;
+    }
+
+    public Boolean getAI() {
```
public Boolean getIsAI(){

gitonga13 commented:
Went ahead and added the getters and setters

maxschlesinger commented:
This code has been merged to master. Will open up PR to `develop` and add comments back in to be addressed.

maxschlesinger commented:
This PR and branch will be closed. The PR will remain for the comments that need to be addressed.

gitonga13 commented:
Went ahead and made the requested changes: 
1. Added a TODO in the getter for score to call the mod function.
2. Rename roll1 and roll2 to dice1 and dice2.
3. Added name assignment, if isAI is true then name = "CPU" else "Player"
